ansible/roles/nessus/files/nessus_base.py

```diff
@@ -0,0 +1,177 @@
+#!/usr/bin/env python
+
+import sys
+import datetime
+import copy
+import glob
+import time
+import logging
+import ssl
+import json
+import requests
+import copy
+import os
+
```

Comment:
I personally like imports to be in alphabetical order, so I can quickly find the one I'm looking for or determine that one is missing.

Reply:
If only OCD were the extent of my problems, @KyleEvers!